packages/dx-react-core/src/index.ts

```diff
@@ -9,6 +9,9 @@ export { TemplateConnector } from './plugin-based/template-connector';
 /** @internal */
 export { Draggable } from './draggable';
 
+/** @internal */
+export { clear } from './draggable/selection-utils';
```

Comment:
In a context of a package it's not clear what `clear` does exactly. Maybe we should rename it to `clearSelection`?